nuxeo-jsf/nuxeo-platform-ui-web/src/main/java/org/nuxeo/ecm/platform/ui/web/util/files/FileUtils.java

```diff
@@ -154,18 +154,11 @@ public static String retrieveFilename(Part part) {
      * Returns a clean filename, stripping upload path on client side.
      * <p>
      * Fixes NXP-544
+     *
+     * @deprecated since 9.1. Use {@link org.nuxeo.common.utils.FileUtils#getSanitizedFilename(String)}.
      */
     public static String getCleanFileName(String filename) {
-        String res = null;
-        int lastWinSeparator = filename.lastIndexOf('\\');
-        int lastUnixSeparator = filename.lastIndexOf('/');
-        int lastSeparator = Math.max(lastWinSeparator, lastUnixSeparator);
-        if (lastSeparator != -1) {
-            res = filename.substring(lastSeparator + 1, filename.length());
-        } else {
-            res = filename;
-        }
-        return res;
+        return org.nuxeo.common.utils.FileUtils.getSanitizedFilename(filename);
```

Comment:
The main goal of this `getCleanFileName` is to strip any initial path, so for instance `/tmp/2349876398/foo.pdf` is turned into `foo.pdf`. I'm not sure it's worth merging this with the new implementation stripping forbidden chars when creating a File actually. They have different goals (which should be reflected in their Javadocs). Cc @atchertchian